docs/http_api.rst

```diff
@@ -378,7 +424,14 @@ to belong to exactly one organization, but an organization can have many teams.
        HTTP/1.1 200 OK
        Content-Type: application/json
 
-       {"title":"Brotherhood Without Banners","id":4,"teams":[],"url":"https://example.org/organizations/4","users":[]}
+       {
+        "title":"Brotherhood Without Banners",
+        "id":4,
+        "teams":[],
+        "url":"https://example.org/organizations/4",
```

Comment:
Initially, I would have voted for a trailing slash for collections and no trailing slash for resources. Unless we make our own router though, it does imply redirects for the cases where we don't have trailing slashes though, since that seems to be the default behaviour for the router we use. Since that isn't ideal, I'm less against us just adding trailing slashes for all endpoints.